docs/src/main/tut/entity.md

```diff
@@ -95,8 +95,8 @@ If you're working with [twirl] templates, there's a bridge for that too:
 - scala-twirl: `"org.http4s" %% "http4s-twirl" % Http4sVersion`
 
 [streams-tutorial]: https://gist.github.com/djspiewak/d93a9c4983f63721c41c
-[EntityEncoder]: http://http4s.org/api/0.15/#org.http4s.EntityEncoder$
-[EntityDecoder]: http://http4s.org/api/0.15/#org.http4s.EntityDecoder$
-[MediaType]: http://http4s.org/api/0.15/#org.http4s.MediaType$
-[MediaRange]: http://http4s.org/api/0.15/#org.http4s.MediaRange$
+[EntityEncoder]: http://http4s.org/v0.15/api/#org.http4s.EntityEncoder$
+[EntityDecoder]: http://http4s.org/v0.15/api/#org.http4s.EntityDecoder$
+[MediaType]: http://http4s.org/v0.15/api/#org.http4s.MediaType$
+[MediaRange]: http://http4s.org/v0.15/api/#org.http4s.MediaRange$
```

Comment:
If we make these relative, they'll work in v0.16 and beyond: `../api/`.  This is one of the factors that motivated the switch from `api/vX.Y` to `vX.Y/api`.